src/app/containers/Headings/index.stories.jsx

```diff
@@ -0,0 +1,15 @@
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import React from 'react';
+import HeadingsContainer from '.';
+import deepGet from '../../helpers/json/deepGet';
+import articleData from '../../../../data/test/news/articles/c0000000007o.json';
```

Comment:
It'd be better to use the headline/subheadline blocks that are used in the unit tests https://github.com/bbc/simorgh/blob/latest/src/app/containers/Headings/index.test.jsx rather than the article data. This is because the JSON in the `data` can be changed for multiple reasons and we want these containers to have storybook examples that are self-contained.

Reply:
done 😄